pkg/syncer/k8scloudoperator/placement.go

```diff
@@ -186,6 +190,11 @@ func isSPInList(name string, spList []StoragePoolInfo) bool {
 	return false
 }
 
+func stampPVCWithError(ctx context.Context, client kubernetes.Interface, curPVC *v1.PersistentVolumeClaim, errAnnotation string, err error) error {
```

Comment:
err supplied as argument in `stampPVCWithError` func is not used, and just returned back to the caller. If not required can we remove `err` argument?